packages/ember-auto-import/ts/package.ts

```diff
@@ -127,17 +127,26 @@ export default class Package {
     // is using. We will use these so we can configure our parser to
     // match.
     let babelAddon = instance.addons.find(addon => addon.name === 'ember-cli-babel') as any;
-    let babelOptions = babelAddon.buildBabelOptions(options);
-    let extensions = babelOptions.filterExtensions || ['js'];
+    let version = parseInt(babelAddon.pkg.version.split('.')[0], 10);
+    let babelOptions, extensions;
+
+    if (typeof babelAddon.getSupportedExtensions === 'function') {
+      babelOptions = babelAddon.buildBabelOptions('babel', options);
```

Comment:
I added this new signature so that I could make the default invocation without arguments (or options) passed return the broccoli-babel-transpiler style options.